app/assets/stylesheets/global.scss

```diff
@@ -1,3 +1,40 @@
+@font-face {
+  font-family: "cityscape";
+  src: asset-url("cityscape");
+}
+
+@font-face {
+  font-family: "lethal_league";
+  src: asset-url("Lethal_League");
+}
+
+@font-face {
+  font-family: "will-harris-reg";
+  src: asset-url("Will-Harris-Regular");
+}
+
+@font-face {
+  font-family: "hometown";
+  src: asset-url("Hometown");
+}
+
+@font-face {
+  font-family: "assassin";
+  src: asset-url("Assassin");
+}
+
+@font-face {
+  font-family: "romance";
+  src: asset-url("romance");
+}
+
+@font-face {
```

Comment:
So after doing some digging it looks like this is something new I haven't seen before :) It's actually loading the font each time you do this, sort of like calling a font-face method for each of the fonts you're importing. So it's not setting it to a global variable, its performing a rule for each of the fonts. So this code is actually all set :)